src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests.cs

```diff
@@ -14178,6 +14178,56 @@ public void Add(T t) { }
             CompileAndVerify(comp, expectedOutput: "[],");
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/75560")]
+        public void Nullable_Spread_01()
+        {
+            var source = """
+                #nullable enable
+                using System.Collections.Generic;
+                IEnumerable<object?> a = [null];
+                object[] b = [..a];
+                object[] c = [..a!];
```

Comment:
>a [](http://example.com/codeflow?start=32&length=1)

I would strengthen the tests and used a different local here in order to avoid a chance that the error was not reported here because it was reported for the same local on the previous line.